core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -1231,10 +1231,7 @@ impl ListableSubstateDatabase for RocksDBStore {
         Box::new(
             self.open_db_context()
                 .cf(SubstatesCf)
-                .iterate(Direction::Forward)
-                .map(|(iter_key_bytes, _)| iter_key_bytes.0)
-                // Rocksdb iterator returns sorted entries, so ok to to eliminate
-                // duplicates with dedup()
+                .iterate_key_groups()
                 .dedup(),
```

Comment:
I don't think we need the dedup any more, right?

Reply:
ahh yes, neither the `Box::new()` :)